packages/lesswrong/client/logging.ts

```diff
@@ -72,7 +72,7 @@ window.addEventListener('load', ev => {
     performance: {
       memory: (window as any).performance?.memory?.usedJSHeapSize,
       timeOrigin: window.performance?.timeOrigin,
-      timing: window.performance?.timing,
+      timing: window.performance?.timing?.toJSON?.(),
```

Comment:
`window.performance.timing` is an instance of a class which later on gets passed to `JSON.stringify`. This should be exactly the same on all platforms that matter (some less conformant platforms might not stringify properly, but such platforms almost certainly don't implement `window.performance.timing` anyway), but now the type system is happy.